contrib/automation_tests/orbit_symbol_cache.py

```diff
@@ -0,0 +1,62 @@
+"""
+Copyright (c) 2021 The Orbit Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+"""
+
+from absl import app
+
+from core.orbit_e2e import E2ETestSuite
+from test_cases.connection_window import FilterAndSelectFirstProcess, ConnectToStadiaInstance
+from test_cases.symbols_tab import LoadAllSymbolsAndVerifyCache, ClearSymbolCache, LoadSymbols, ReplaceFileInSymbolCache
+from test_cases.main_window import EndSession
+
+"""Test symbol loading with and without local caching.
```

Comment:
nit. Usually the comments are formatted like this
```
"""
Test symbol ...
```

Did the python formatter produce this?

Reply:
No, that was copy & paste from another test... I adjusted it!